src/main/java/io/vertx/core/loadbalancing/LoadBalancer.java

```diff
@@ -64,6 +69,66 @@ public <E> Endpoint<E> endpointOf(E endpoint) {
     }
   };
 
+  /**
+   * Random load balancer.
+   */
+  // ThreadLocalRandom? contention, seed?
+  LoadBalancer RANDOM = () -> {
+    Random random = new Random();
+    return (EndpointSelector) endpoints -> {
+      if (endpoints.isEmpty()) {
+        return -1;
+      }
+      return random.nextInt(endpoints.size());
+    };
+  };
+
+  /**
+   * Power of two choices load balancer.
+   */
+  // BEST_OF_TWO? P2C?
+  LoadBalancer POWER_OF_TWO_CHOICES = () -> new EndpointSelector() {
+    final Random random = new Random();
+
+    private List<Integer> sample(int length) {
```

Comment:
I implemented Robert Floyd's here, I couldn't find any existing util for this. Did I miss sth?
Should we move it?

Reply:
I think you could simplify this, take a look at the implementation in Stork https://github.com/smallrye/smallrye-stork/blob/09c4b19c14993dc164a799a1c84f1dc5e9cb695c/load-balancer/power-of-two-choices/src/main/java/io/smallrye/stork/loadbalancer/poweroftwochoices/PowerOfTwoChoicesLoadBalancer.java#L34

Reply:
I'm afraid they're not implementing the algorithm as is; there is no guarantee that the two picked instances are different. I don't know the consequences of that.
But if you want to keep it "simple", we can do the same here.

Reply:
yes, Stork's impl might not inspect two different endpoints every time. But then you could use a `do/while` to select a random int for the second endpoint until it's different than the first one.

Reply:
Was my first choice, not as efficient as Floyd's, but it's simpler.

Reply:
Perhaps, as a follow-up PR we can examine how one implementation performs compared to the other, using a JMH test.